wls-wahlvorstand-service/src/test/resources/http.request/http-client.env.json

```
@@ -0,0 +1,14 @@
+{
+  "docker": {
+    "token_type": "",
+    "auth_token": "",
+    "WLS_WAHLVORSTAND_SERVICE_URL": "http://localhost:8201",
+    "SSO_URL": "http://kubernetes.docker.internal:8100"
+  },
+  "nonDocker": {
+    "token_type": "",
+    "auth_token": "",
+    "WLS_WAHLVORSTAND_SERVICE_URL": "http://localhost:39155",
+    "SSO_URL": "http://localhost:8100"
```

Sonst funktionieren die Requests nicht mit der Security

```suggestion
    "SSO_URL": "http://kubernetes.docker.internal:8100"
```